### PR TITLE
Deprecates AequilibraEData in favor of Pandas DataFrames

### PR DESCRIPTION
**Pull request by pedrocamargo**

### CODE REVIEW

**Comment by Jake-Moss** [comment]
Just trying this out locally now, there's a couple failures but nothing severe. I do think we should sort out the pandas specific warnings, I got about 40 while running the tests, ~12  of those are from pandas

File: aequilibrae/matrix/aequilibrae_matrix.py [hunk 1]
```diff
@@ -283,8 +283,8 @@ def get_matrix(self, core: str, copy=False) -> np.ndarray:
 
     def create_from_omx(
         self,
-        file_path: str,
         omx_path: str,
+        file_path: str = None,
```
Comment (Jake-Moss) [inline aequilibrae/matrix/aequilibrae_matrix.py:287]: This is actually a breaking change, people who relied on the position of the arguments (us, in our docs) will have the file paths swapped

**Review by Jake-Moss — APPROVED** [review]
Looks like CI is happy with me now. Changes look good to me

File: aequilibrae/matrix/aequilibrae_matrix.py (hunk 1 above)
Comment (pedrocamargo) [reply aequilibrae/matrix/aequilibrae_matrix.py:287]: True, but there is not such way out, as the best behaviour is to have it as a memory-only matrix.

I would doubt anyone uses this, however.

**Pull request merged**